sidebery.json

```diff
@@ -2,19 +2,61 @@
   "settings": {
     "nativeScrollbars": true,
     "nativeScrollbarsThin": true,
-    "ctxMenuNative": true,
-    "navBarLayout": "hidden",
+    "updateSidebarTitle": true,
+    "ctxMenuNative": false,
```

Comment:
Was there any specific reason to use the non-native context menu by default? I think the default option should probably be to use the native menu as it seemed to work better most of the time.

Reply:
There is an [issue](https://github.com/mbnuqw/sidebery/issues/1816) with the way I use Sidebery. In any case, I have removed settings that may be specific to the way I use Sidebery.